package.json

```diff
@@ -42,7 +43,7 @@
   },
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "tsc && oclif-dev manifest && oclif-dev readme",
+    "build": "tsc --module commonjs && oclif-dev manifest && oclif-dev readme",
```

Comment:
This part `oclif-dev manifest && oclif-dev readme` should be a `postbuild` step

Reply:
There is no `postbuild` step.

Reply:
he's saying there _could_ be. Every script (even custom ones) support `pre` and `post` scripts.

@XVincentX  Is there an advantage to separating it into a build and a postbuild script?

Reply:
The reason I prefer it is that if the script encounters an error it can be difficult to see which part of it failed.

If you have separate commands, you get the precise error.